README.md

```diff
@@ -1,77 +1,373 @@
+MacOS; `rm -r NameOfDirectory`
+Windows: `rmdir /S NameOfDirectory`
+
+### Code Editor
+
+An Integrated Development Environment (IDE) is a software application that provides comprehensive tools and features to assist developers in writing, debugging, and deploying software. An IDE typically includes a code editor, a debugger, a compiler or interpreter, and other tools such as a profiler, a version control system, and a build automation tool.
+An IDE can also provide features like auto-completion, syntax highlighting, and code refactoring to make coding faster and more efficient. IDEs are commonly used for developing software applications, web applications, and mobile apps. Our preferred IDE is Visual Studio Code. Download it [here](https://code.visualstudio.com/). You can also install our developer extension pack by going to extensions and typing [VetsWhoCode Extension Pack](https://marketplace.visualstudio.com/items?itemName=VetsWhoCode.vetswhocode-extension-pack).
+
+### Browser
+
+The preferred browser for Vets Who Code is the Google Chrome, due to its popularity and its robust developer tools. To download it please go [here](https://www.google.com/chrome/) and follow the prompts.
+
+
+### Using GitHub
+ We use GitHub extensively here. Below you’ll find the steps to use it within the terminal or command prompt.
+
+1.	 Install Git: Git is a version control system used by GitHub. You can download and install Git for Windows from the official Git website.
+2.	 Configure Git: After installing Git, open the Git Bash application and configure your name and email address using the following commands:
+```console
+git config --global user.name "Your Name"
+git config --global user.email "youremail@example.com"
+```
+
+
+
+3.	Create a repository: A repository is a place where you can store and manage your code. Click on the “New” button and create a new repository.
+4.	Clone the repository: To start working with the repository, you need to clone it to your local machine. You can do this by clicking on the “Code” button and copying the repository URL. Then, open your terminal or command prompt and type `git clone URL-Goes-Here`.
+5.	Make changes to the repository: Once you have cloned the repository, you can change the code using your favorite code editor.
+6.	Add changes to the staging area: To add your changes to the staging area, type `git add` followed by the file name or directory you want to add. You can use `git add .` to add all the changes.
+7.	Commit changes: Once you have added your changes to the staging area, you can commit them by typing `git commit -m` followed by a brief description of the changes you made.
+8.	Push changes: To upload your changes to the GitHub repository, type `git push` followed by the name of the branch to which you want to push the changes.
```

Comment:
```suggestion
8.	Push changes: To upload your changes to the GitHub repository, type `git push`. If you're branch isn't setup check the output in the command line it may have a message like `git push --set-upstream origin <branch name>`. Once you've setup your upstream, you can use `git push` for all future uploads to this branch.
```